oneflow/api/cpp/framework/tensor.h

```diff
@@ -37,9 +37,18 @@ class Tensor final {
   explicit Tensor(const Shape& shape = Shape(), const Device& device = Device("cpu"),
                   const DType& dtype = DType::kFloat);
   explicit Tensor(const std::shared_ptr<oneflow::one::Tensor>& tensor);
-  [[nodiscard]] const Shape shape() const;
-  [[nodiscard]] const Device device() const;
-  [[nodiscard]] const DType dtype() const;
+
+  Tensor(const Tensor& tensor);
+  Tensor(Tensor&& tensor) noexcept;
+
+  ~Tensor();
```

Comment:
```suggestion
  ~Tensor() = default;
```
然后可以删去 .cpp 内的 `~Tensor()`